src/test/scala/com/redislabs/provider/redis/rdd/RedisSqlStandaloneSuite.scala

```diff
@@ -47,10 +51,113 @@ class RedisSqlStandaloneSuite extends FunSuite with ENV with BeforeAndAfterAll w
     loadedArr.sortBy(_.name) should be(data.toArray.sortBy(_.name))
   }
 
+  test("overwrite data when it's empty") {
+    // generate random table, so we can run test multiple times and not append/overwrite data
+    val tableName = "person" + UUID.randomUUID().toString.replace("-", "")
```

Comment:
should we extract this table name generation into a function in this Spec and reuse between tests?